### PR TITLE
subnet: externalipam_group support

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
@cmeissner based on https://github.com/theforeman/foreman/pull/8067 I think this should be enough to get the group feature working after all… could you please test in your env?

File: plugins/module_utils/foreman_helper.py [hunk 1]
```diff
@@ -92,10 +92,12 @@ def inner(self, *args, **kwargs):
     return decor
 
 
-def _check_patch_needed(fixed_version=None, plugins=None):
+def _check_patch_needed(introduced_version=None, fixed_version=None, plugins=None):
     """
     Decorator to check whether a specific apidoc patch is required.
 
+    :param introduced_version: The version of Foreman the API bug was introduced.
+    :type introduced_version: str, optional
```
Comment (mdellweg) [inline plugins/module_utils/foreman_helper.py:107]: Somehow i was missing this, but didn't want to ask...

**Review by mdellweg — APPROVED** [review]
CI looks like a galaxy hickup.

File: plugins/module_utils/foreman_helper.py (hunk 1 above)
Comment (evgeni) [reply plugins/module_utils/foreman_helper.py:107]: My very first version of that had it, but then I was like "fuck, we'll never need it". Well, never didn't last a week.

**Comment by evgeni** [comment]
I'd like to wait with the merge until @cmeissner has a chance to verify the APIdoc patch is correct.

**Review by ehelms — APPROVED** [review]

**Comment by cmeissner** [comment]
@evgeni unfortunately I ran still in this error:

```... Unsupported parameters for (theforeman.foreman.subnet) module: externalipam_group ...```

Are there any needs of updating components. I only installed the latest version (1.4.0) of the glory ansible modules.

**Comment by mdellweg** [comment]
> @evgeni unfortunately I ran still in this error:
> 
> `... Unsupported parameters for (theforeman.foreman.subnet) module: externalipam_group ...`
> 
> Are there any needs of updating components. I only installed the latest version (1.4.0) of the glory ansible modules.

Well, it's not even merged to develop, so you would need to test with this feature branch.

I theory, this should work, https://docs.ansible.com/ansible/latest/user_guide/collections_using.html#installing-a-collection-from-a-git-repository

I think the best option is to get a local copy of this repository, check out evgeni's branch and run make dist. You can then install the collection from the generated tar.gz or point your ansible.cfg to the build/collections directory (see https://github.com/theforeman/foreman-ansible-modules/blob/develop/ansible.cfg#L2).

Whatever works for you should be a good addition to:
https://github.com/theforeman/foreman-ansible-modules/tree/develop/docs#installation

But you can also "just" apply the diff as a patch to your installed version, or copy the files there.

**Review by cmeissner — COMMENTED** [review]
With ansible modules build from feature branch it works as expected. ﻿

**Comment by evgeni** [comment]
Perfect, thanks for testing!

**Comment by mdellweg** [comment]
> With ansible modules build from feature branch it works as expected.

/me `make dance`

**Pull request merged**